### DISCUSSION TITLE
rubiidev needs to fix this

### DISCUSSION THREAD

**Discussion by mariocraft987** [cytrinc/bark-coding#196] [General]
@Rubiidev-18 needs to fix #183

**Comment by raynepaws** [comment]
i believe the `pointdir` block is to blame for this madness

**Comment by raynepaws** [comment]
why rubiidev? it's not their fault. and why isn't blockly recognizing field values, what did we change? @Bashamega do you think you might've accidentally changed something? the js isn't recognizing the `fieldColour` or `fieldAngle` field types

**Reply by Bashamega** [reply]
> why rubiidev? it's not their fault. and why isn't blockly recognizing field values, what did we change? @Bashamega do you think you might've accidentally changed something? the js isn't recognizing the `fieldColour` or `fieldAngle` field types

I don't think so.
I didn't play with the files
